add-on/src/lib/ipfs-client/index.js

```diff
@@ -26,18 +26,24 @@ async function destroyIpfsClient () {
       await client.destroy()
     } finally {
       client = null
-      await _reloadIpfsClientDependents()
+      await _reloadIpfsClientDependents() // sync (API stopped working)
     }
   }
 }
 
+function _isWebuiTab (url) {
+  const bundled = !url.startsWith('http') && url.includes('/webui/index.html#/')
+  const ipns = url.includes('/webui.ipfs.io/#/')
```

Comment:
@olizilla FYI this will detect webui loaded from:
- `https://webui.ipfs.io`
- `http://127.0.0.1:8080/ipns/webui.ipfs.io`

Reply:
`#/` will only be added by the app after the initial bundle has loaded (unless the user has bookmarked it explicitly)...

Reply:
Yes, that was on purpose to remove false-positive matches on random websites and avoid double-reload when quickly switching between backends.